diff --git a/setup/projects/boilerplate/article.js b/setup/projects/boilerplate/article.js
index 9d851a7..e4e1e6e 100644
--- a/setup/projects/boilerplate/article.js
+++ b/setup/projects/boilerplate/article.js
@@ -21,12 +21,21 @@ module.exports = {
     name: 'title'
   }, {
     name: 'paragraph'
+  }, {
+    name: 'header'
   }, {
     name: 'image'
   }, {
     name: 'tweet'
   }],
 
+  defaultContent: [{
+    identifier: 'header',
+    position: 'fixed'
+  }, {
+    identifier: 'paragraph'
+  }],
+
   metadata: [{
     handle: 'title',
     type: 'li-text',
diff --git a/setup/projects/boilerplate/boilerplate_config.js b/setup/projects/boilerplate/boilerplate_config.js
index b658c74..099f895 100644
--- a/setup/projects/boilerplate/boilerplate_config.js
+++ b/setup/projects/boilerplate/boilerplate_config.js
@@ -10,7 +10,8 @@ module.exports = {
     require('./components/title'),
     require('./components/paragraph'),
     require('./components/image'),
-    require('./components/tweet')
+    require('./components/tweet'),
+    require('./components/header')
   ],
   categories: require('./categories')
 }
diff --git a/setup/projects/boilerplate/design_settings.js b/setup/projects/boilerplate/design_settings.js
index 1754211..def6644 100644
--- a/setup/projects/boilerplate/design_settings.js
+++ b/setup/projects/boilerplate/design_settings.js
@@ -10,7 +10,7 @@ module.exports = {
   componentGroups: [{
     name: 'text',
     label: 'Text',
-    components: ['title', 'paragraph']
+    components: ['title', 'paragraph', 'header']
   }, {
     name: 'media',
     label: 'Medien',
